tests/Makefile.am

```diff
@@ -2,17 +2,7 @@
 # Copyright © 2018 VMware, Inc. All Rights Reserved.
 
 AM_CPPFLAGS = -I. -I$(top_srcdir)/src
-
-bin_PROGRAMS = test_api tst-condpi2 tst-cond1
-
-test_api_SOURCES = test_api.c rtpi.h
```

Comment:
By removing all references to rtpi.h, and not adding it as an explicit dependency for the test targets - doesn't this break triggering a rebuild when nothing but rtpi.h changes?

Reply:
I can confirm experimentally that any change in rtpi.h will trigger a rebuild when running the 'make test' (i.e. 'make check') target.

Would that be a sufficient test? I'm attempting to map out the dependency chain that leads to the rebuild but automake generated makefiles or not the easiest read.